### PR TITLE
Julia updates

### PR DESCRIPTION
**Pull request by Lizzydapsy**
Please review: Information on Julia softwareto be added to the HPC document

### CODE REVIEW

File: sharc/software/apps/julia.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.. _julia_sharc:
+
+Julia
+=====
+
+.. sidebar:: JULIA
```
Comment (willfurnass) [inline sharc/software/apps/julia.rst:6]:
```suggestion
.. sidebar:: Julia
```

File: sharc/software/apps/julia.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+Installation notes
+------------------
+These are primarily for administrators of the system.
+
+**julia version 1.8.2**
+
+* wget https://julialang-s3.julialang.org/bin/linux/x64/1.8/julia-1.8.2-linux-x86_64.tar.gz
+* tar -xf julia-1.8.2-linux-x86_64.tar.gz
```
Comment (willfurnass) [inline sharc/software/apps/julia.rst:33]: Indent like so to get fixed-with font:
```suggestion
   wget https://julialang-s3.julialang.org/bin/linux/x64/1.8/julia-1.8.2-linux-x86_64.tar.gz
   tar -xf julia-1.8.2-linux-x86_64.tar.gz
```

File: sharc/software/modulefiles/apps/julia/1.8.2/binary [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+        puts stderr " Makes julia $ver available to the system."
+}
+
+# julia version (not in the user's environment)
+set     ver     1.8.2
+
+module-whatis   "sets the necessary julia $ver paths"
+
+prepend-path PATH /usr/local/packages/apps/julia/1.8.2/binary/bin
```
Comment (willfurnass) [inline sharc/software/modulefiles/apps/julia/1.8.2/binary:18]:
```suggestion
set ver       1.8.2
set juliaroot /usr/local/packages/apps/julia/1.8.2/binary

module-whatis   "sets the necessary julia $ver paths"

prepend-path PATH              $juliaroot/bin
prepend-path CMAKE_PREFIX_PATH $juliaroot 
prepend-path CPATH             $juliaroot/include 
prepend-path LD_LIBRARY_PATH   $juliaroot/lib 
prepend-path LIBRARY_PATH      $juliaroot/lib 
prepend-path MANPATH           $juliaroot/share/man 
prepend-path XDG_DATA_DIRS     $juliaroot/share 
```

File: sharc/software/apps/julia.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+Interactive Usage
+-----------------
+After connecting to sharc (see :ref:`ssh`),  start an interactive session with the 
+:ref:`qrshx` or :ref:`qrsh` command. 
+
+The latest version of Julia (currently 1.8.2) is made available with the command ::
+
+        module load apps/julia/1.8.2/binary
+
+You can then run the command line version by entering ``julia``.
```
Comment (willfurnass) [inline sharc/software/apps/julia.rst:23]: Julia is _only_ command-line so how about the following instead?

```suggestion
You can then start Julia with ``julia``.
```

**Comment by Lizzydapsy** [comment]
@willfurnass, thank you for your suggestion. Surprsingly, these + signs were not directly obvious in the VS code text. Also it seems not to have any effect on the pre-built documentation. I will make the changes nevertheless.

**Comment by desmfryan** [comment]
Hi Elizabeth

Looks good apart from the install instructions bit.

If you look at other docs (e.g. Lammps) the install commands should be in a script. Also, there should be some words like: 
Julia was installed using 'link to install file' script. Also the commands you included are not complete.....

Look at a few other the other softwares to see what you need to do and ensure that the install script file is correct i.e. if I run this this it will fully install julia.

Thanks.

Des

**Comment by Lizzydapsy** [comment]
@desmfryan, actioned. Thank you.

**Comment by desmfryan** [comment]
Hi Elizabeth

Looks good to merge.

Cheers,

Des

**Pull request merged**